fuel-vm/src/util.rs

```diff
@@ -449,11 +503,17 @@ pub mod test_helpers {
             &mut self,
             checked: Checked<Create>,
         ) -> anyhow::Result<StateTransition<Create>> {
-            let mut transactor = Transactor::new(
-                self.storage.clone(),
-                self.params,
-                self.gas_costs.clone(),
-            );
+            let interpreter_params = InterpreterParams {
```

Comment:
It is better if `Self` contains `InterpreterParams` instead=)

Reply:
I added the `ConsesensusParams` to `Self` and used the `into`.